.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "kibana-proxy"]
 	path = kibana-proxy
 	url = https://github.com/fabric8io/openshift-auth-proxy.git
+[submodule "lib/openshift-ansible"]
```

Comment:
I'm hoping submodule allows us to peg to the ansible version, but ultimately I think it probably is dependent upon how being able to test a PR here and one one there at the same time.  I don't really know whats best here. @stevekuznetsov are you able to offer insight?